src/index.html

```diff
@@ -10,7 +11,39 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <label class="label page__body__label-big">
+      <div class="label__search-icon label__search-icon--big"></div>
+      <form
+      data-qa="big"
+      action="af"
+      method="post"
+      class="form page__body__label__form"
+      >
```

Comment:
Need to fix it everywhere
```suggestion
      <form
        data-qa="big"
        action="af"
        method="post"
        class="form page__body__label__form"
      >
```